crates/sui-graphql-rpc/tests/snapshots/snapshot_tests__test__schema_sdl_export.snap

```diff
@@ -551,8 +551,8 @@ input TransactionBlockFilter {
 }
 
 enum TransactionBlockKindInput {
-	PROGRAMMABLE_TX
```

Comment:
Can we keep them in alphabetical order?

Reply:
yeah I think I can do that, I just needed SYSTEM_TX=0 and PROGRAMMABLE_TX=1, but I think I can just assign that to each enum variant